templates/primary-secondary/primary-secondary.js

```diff
@@ -936,12 +933,14 @@ class TemplatePrimarySecondary extends FocusVisiblePolyfillMixin(RtlMixin(LitEle
 		const scrollClasses = {
 			'd2l-template-scroll': isWindows
 		};
+		const keyboardHelpText = this._isMobile ? this.localize(`${this._namespace}.keyboardVertical`) : this.localize(`${this._namespace}.keyboardHorizontal`);
```

Comment:
As handy as it is, can you not use `_namespace` and just use the full string? It helps when we're doing searches for terms to see if they're used if we can search for the full term name.

Reply:
Sure, I was following the pattern I'd seen elsewhere in core but I'll switch it up. :)

Reply:
Oh no! Haha now I need to stamp those out.